bacon/model/StorageCouchBaseDB.swift

```diff
@@ -87,6 +87,10 @@ class StorageCouchBaseDB {
         }
     }
 
+    func getNumberOfTransactionsInDatabase() -> Double {
+        return Double(transactionDatabase.count)
+    }
+
```

Comment:
why is it a Double?

Reply:
Because database getCount() method returns a UInt64 by default. Type casting to integer will fail on the 32-bit system.